lib/fathead/mdn_css/fetch.pl

```diff
@@ -67,7 +67,47 @@
 my $current_active_connections = 0;
 my $maximum_active_connections = 4;
 
-my @keyword_urls = keys %urls;
+#save the urls with fragments to a text file called
+#fragments.txt so that parse.pl can use this information
+#to extract extra information about the fragments
+my @keyword_urls        = map  { Mojo::URL->new($_) } keys %urls;
+my @urls_with_fragments = grep { $_->fragment } @keyword_urls;
+if (@urls_with_fragments) {
+    open( my $fh, '>:encoding(UTF-8)', catfile 'download', 'fragments.txt' )
+      or die $!;
+
+=begin
+    For fragment in urls past this path transform-function we can get
+    the link for the actual fragment in the following way.
+    Take for example this fragment #matrix()
+    In order to get the actual link, we take the fragment, matrix(), remove
+    the brackets, and append it to the original path so that we get the link
+    https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix
+=cut
+
+    for my $url_with_fragment (@urls_with_fragments) {
+        if ( $url_with_fragment =~ qr/transform-function/ ) {
```

Comment:
@hchienjo why are we only grabbing the `transform-functions` page here?

Shouldn't we grab any page that has fragments? e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/filter#blur()